static/js/message_edit_history.js

```diff
@@ -51,6 +50,11 @@ export function fetch_and_render_message_history(message) {
                     item.topic_edited = true;
                     item.prev_topic = msg.prev_topic;
                     item.new_topic = msg.topic;
+                } else if (msg.prev_stream) {
+                    item.posted_or_edited = "Stream edited by";
```

Comment:
I notice these strings are not tagged for translation, which is a bug. Can you add a new commit to this PR that adds translation tags to the `posted_or_edited` strings here?

Reply:
Will do that